Yoast/javascript
# Add irregular plurals to Italian function words

## Summary


This PR can be summarized in the following changelog entry:

* adds some plural forms of Italian function words

## Test instructions

This PR can be tested by following these steps:

* Check the unit tests:
 Run yarn test and make sure all tests still pass

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-435
